docs/user_guide/preemption.md

```diff
@@ -228,4 +228,31 @@ In this example, two imbalances are observed:
 | `rt.ten-a.queue-2` | 0                          | 0                         |
 | `rt.ten-b`         | 15                         | 10                        |
 | `rt.ten-b.queue-3` | 15                         | 10                        |
-| `rt.sys`           | 0                          | 10                        |
\ No newline at end of file
+| `rt.sys`           | 0                          | 10                        |
+
+### Redistribution of Quota and Preemption Storm
+
+#### Redistribution of Quota
+
+Setting up guaranteed resources for the queue present at a higher level in the whole queue hierarchy helps to re-distribute the quota among different groups especially when workloads of the same priority run in different groups. Unlike the default scheduler, Yunikorn preempts even the workloads of the same priority to free up resources for pending workloads who deserve to get the resources as per guaranteed quota. At times, one needs this kind of queue set up in a real production cluster for redistribution.
+
+For example, root.region[1-N].country[1-N].state[1-N]
+
+![preemption_quota_redistribution](../assets/preemption_quota_redistribution.png)
+
+This queue set up has N regions under “root”, each region has N countries. If administrators want to redistribute the workloads of the same priority among different regions, then it is better to define the guaranteed quota for each region so that preemption helps to reach the situation of running the workloads by redistribution based on the guaranteed quota each region is supposed to get. That way each region uses the resources it deserves to get at the maximum possible level from the overall cluster resources.
+
+#### Preemption Storm
+
+With setup like above, there is a side effect of increasing the possibilities of preemption storm or loop happening within the specific region between different state queues (siblings belonging to same parent).
```

Comment:
add "a" to "With a setup"
replace "possibilities" with "chance"
add "a" to "a preemption storm or loop"